### PR TITLE
Attempting to fix more flakey auth0 cli tests

### PR DESCRIPTION
**Pull request by jamescrowley**
- [x] Ordering not guaranteed from auth0 (failed here: https://app.circleci.com/pipelines/github/boxwise/boxtribute/5143/workflows/e7218845-2c9b-42d5-af25-5518241f4ab4/jobs/28085/tests#failed-test-0)
- [x] Assertion of roles (failed here: https://app.circleci.com/pipelines/github/boxwise/boxtribute/5131/workflows/5491e010-baf1-4b44-99a4-897fae266a9b/jobs/28004)

### CODE REVIEW

**Comment by jamescrowley** [comment]
As noted in Slack @pylipp the more I look at this, none of my modifications will address the root cause of this flakey auth0 CLI tests. However, maybe this will reduce the incidence while we figure out a less brittle approach.

File: back/test/integration_tests/test_cli.py [hunk 1]
```diff
@@ -320,10 +322,13 @@ def test_remove_base_access(patched_input, mysql_data, auth0_management_api_clie
...
     }
+
+    # verify the roles still exist for the other bases
+    auth0_roles, _ = mysql_data
     role_ids = auth0_management_api_client.get_single_base_user_role_ids(base_id)
-    assert len(role_ids) == 1
+    assert auth0_roles[f"base_{base_id}_volunteer{test_role_name_suffix}"]['id'] in role_ids
     role_ids = auth0_management_api_client.get_single_base_user_role_ids(80)
-    assert len(role_ids) == 1
+    assert auth0_roles[f"base_80_volunteer{test_role_name_suffix}"]['id'] in role_ids
```
Comment (jamescrowley) [inline back/test/integration_tests/test_cli.py:331]: @pylipp I'm not sure if this actually still achieves what you wanted to test here so let me know if not!

**Comment by jamescrowley** [comment]
@pylipp this seems to work - it doesn't work if you re-run multiple times because the pipeline id doesn't change, but it works when you queue multiple times (which is the only scenario we'll hit from multiple commits close together time wise or multiple PRs opening)

**Comment by pylipp** [comment]
[2.0 Make BE integration tests run sequentially in CI](https://trello.com/c/LE1E5jNQ/1486-20-make-be-integration-tests-run-sequentially-in-ci)

**Comment by pylipp** [comment]
thx  @jamescrowley :tada: glad to have this integrated! Only smaller nits fixed from my side

**Pull request merged**